manticore/platforms/evm.py

```diff
@@ -2539,6 +2548,12 @@ def constraints(self, constraints):
         if self.current_vm:
             self.current_vm.constraints = constraints
 
+    def last_ins_was_true_jumpi(self) -> Optional[bool]:
+        return self._last_ins_was_true_jumpi
+
+    def set_last_ins_was_true_jumpi(self, jump_cond: Optional[bool]):
```

Comment:
Let's convert this getter/setter pairing to a property as well.

Reply:
Fixed